alphabot/memory.py

```diff
@@ -40,3 +43,18 @@ def _save(self, key, value):
     @gen.coroutine
     def _get(self, key, default):
         raise gen.Return(self.values.get(key, default))
+
+
+class MemoryRedis(Memory):
+    """Redis storage."""
+
+    def __init__(self, host='localhost', port=6379, db=0):
```

Comment:
This won't work. Check out the way the bot engine invokes this. It's generic to a degree where nothing gets passed into the constructor. Convert all these babies into environment variables, and raise an exception if they are missing / can't connect.